deploy/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "deploy-operator"
+version = "0.1.0"
+edition = "2021"
+authors = ["DatenLord <dev@datenlord.io>"]
+description = "The xline-operator is a powerful tool designed to automate the process of bootstrapping, monitoring, snapshotting, and recovering an xline cluster on Kubernetes."
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/xline-kv/xline-operator/tree/main/deploy"
+categories = ["operator"]
+keywords = ["kubernetes", "xline", "operator"]
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.71"
+clap = { version = "4.3.4", features = ["derive"] }
+futures = "0.3.28"
+garde = { version = "0.11.2", default-features = false, features = ["derive", "pattern"] }
```

Comment:
If `grade` is only used for testing, please remove it to `[dev-dependencies]`

Reply:
done